compiler-rt/lib/sanitizer_common/sanitizer_platform_limits_posix.h

```diff
@@ -102,8 +102,10 @@ const unsigned struct_kernel_stat_sz = SANITIZER_ANDROID
                                            ? FIRST_32_SECOND_64(104, 128)
 #      if defined(_ABIN32) && _MIPS_SIM == _ABIN32
                                            : FIRST_32_SECOND_64(176, 216);
-#      else
+#      elif !SANITIZER_MUSL
                                            : FIRST_32_SECOND_64(160, 216);
```

Comment:
IMO it would be better to invert the logic. elif SANITIZER_MUSL

Reply:
Done :)